<launch>
  <master auto="start"/>

  <!-- Run the map server -->
  <!-- <node name="map_server" pkg="map_server" type="map_server" args="/home/ubuntu/map/mymap.yaml.yaml"/> -->

  <!--- Run AMCL -->
  <!-- <include file="$(find amcl)/examples/amcl_omni.launch" /> -->

  <!-- Load configuration -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <remap from="cmd_vel" to="mobile_base/commands/velocity"/>
    <rosparam file="$(find iana_navigation)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find iana_navigation)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find iana_navigation)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find iana_navigation)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find iana_navigation)/param/base_local_planner_params.yaml" command="load" />
  </node>
</launch>
